### PR TITLE
Pipeline Fixes

### PR DESCRIPTION
**Pull request by camdnnn**
Was originally used as an example of a PR but was updated to have fixes for GitHub Automation since it was causing issues

### CODE REVIEW

**Review by t-wing11 — APPROVED** [review]
LGTM

File: backend/README.md [hunk 1]
```diff
@@ -31,22 +31,6 @@ If you want to build an _über-jar_, execute the following command:
 
 The application, packaged as an _über-jar_, is now runnable using `java -jar target/*-runner.jar`.
 
-## Creating a native executable
```
Comment (camdnnn) [inline backend/README.md:34]: Example comment

File: backend/README.md [hunk 2]
```diff
@@ -31,22 +31,6 @@ If you want to build an _über-jar_, execute the following command:
...
-
-Or, if you don't have GraalVM installed, you can run the native executable build in a container using: 
-```shell script
-./mvnw package -Dnative -Dquarkus.native.container-build=true
-```
-
-You can then execute your native executable with: `./target/backend-1.2.0-runner`
-
-If you want to learn more about building native executables, please consult https://quarkus.io/guides/maven-tooling.
-
```
Comment (gr812b) [inline backend/README.md:49]: I like this part of the of the code

**Review by gr812b — CHANGES_REQUESTED** [review]
AHHHHHH THIS IS SO BAD NO NONONO PLEASE NOOOO

File: backend/README.md (hunk 1 above)
Comment (hydrowoxy) [reply backend/README.md:34]: Example reply

**Review by gr812b — APPROVED** [review]

**Pull request merged**